products/binaryauthorization/terraform.yaml

```diff
@@ -23,6 +23,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       vars:
         attestor_name: "test-attestor"
         note_name: "test-attestor-note"
+     - !ruby/object:Provider::Terraform::Examples
+      name: "binary_authorization_attestor_kms"
+      primary_resource_id: "attestor"
+      skip_test: true
```

Comment:
This is because we're using a KMS key, right?

Reply:
Yes, no avoid the "KMS keys left behind" problem. This example should be the same as in the custom test provided.